WindowsPerfGUI/Options/WPerfPath.xaml.cs

```diff
@@ -186,5 +194,33 @@ private void WPerfVersionCheckIgnoreChanged(object sender, RoutedEventArgs e)
             WPerfOptions.Instance.WperfVersionCheckIgnore = !shouldEnforceVersionCheck;
             WPerfOptions.Instance.Save();
         }
+
+        private void WperfDefaultPathCheckbox_Click(object sender, RoutedEventArgs e)
+        {
+            bool newValue = !WPerfOptions.Instance.UseDefaultWperfLocation;
+
+            PathInput.IsEnabled = !newValue;
+            SelectDirectoryButton.IsEnabled = !newValue;
+            if (newValue)
+            {
+                PathInput.Text = WperfDefaults.DefaultWperfPath + "\\wperf.exe";
```

Comment:
I think it's "unsafe" (or at least controversial) to combine path like this?
How about using [Path.Combine](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-8.0) ?